README.md

```diff
@@ -44,12 +44,16 @@ or
 **Documentation**: [current release](https://3yourmind.github.io/kotti/) or [current master](https://kotti.netlify.com/)
 
 ```typescript
-// in main.js / entrypoint
+// in main.ts / entrypoint
 import Vue from 'vue'
 import KottiUI from '@3yourmind/kotti-ui'
 import '@3yourmind/kotti-ui/dist/styles.css'
 
-// register all KtComponents globally
+// also make sure to set-up @vue/composition-api as it’s required for newer Kotti features
+import VueCompositionAPI from '@vue/composition-api' // right now, 0.5.0 is recommended
+Vue.use(VueCompositionAPI)
```

Comment:
should we stress on plugin order as a comment? 
also is it verified to be an issue?

Reply:
What do you mean "verified to be an issue"?

Reply:
if we use the composition plugin after using kotti-ui

Reply:
does it actually prove to be a problem

Reply:
Ah, that’s what you mean. It’s at least conceptually a problem and I don’t want to rely on the order not mattering in the global magic land that vue is. I can add a comment for that then I guess.